app/static/app/js/cipe.js

```diff
@@ -235,17 +235,23 @@ function showCommunicationField() {
     }
 }
 
-function showBecalEndDate() {
+function autoCompleteIsPermanetResidentInput() {
```

Comment:
Por qué autocomplete?

Reply:
Porque cuando en el formulario selecciona que es becario de becal, autocompletamos el campo **is permannet resident**  a no y se muestra el datepicker

Reply:
Ok, está bien, podes agregar en inglés esa documentación arriba de la función?, para explicar lo que hace.

Reply:
Done

Reply:
https://github.com/joausaga/cipe/pull/19/commits/ecf36225e04f63bf5eeecb1ecc739b99a3739b2e